index.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
+
 var fs = require('fs');
 var path = require('path');
 var crypto = require('crypto');
+var gulp = require('gulp');
```

Comment:
You stated the exact reason without realizing it. No point in depending on gulp if we can depend on the same module as gulp and not having to depend on all the other baggage gulp comes with.